src/charts/CytoViz/CytoViz.js

```diff
@@ -84,6 +130,23 @@ export const CytoViz = (props) => {
   const changeZoomPrecision = (event, newValue) => {
     setZoomPrecision(newValue);
   };
+  const toggleAllNodesCollapsed = (event) => {
+    if (compoundsApiRef.current && cytoRef.current) {
```

Comment:
I think we could simplify this condition and check only `compoundsApiRef.current` here, because `compoundsApiRef` and `cytoRef` are both set the `initCytoscape` function, but the current function `toggleAllNodesCollapsed` does not use `cytoRef`.
If this is correct, we could also be able to remove the cytoRef variable, it is only used in `initCytoscape` function: we could use the function parameter `cytoscapeRef` instead.